lib/src/voip/utils/wrapped_media_stream.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:webrtc_interface/webrtc_interface.dart';
+
+import 'package:matrix/matrix.dart';
+import 'package:matrix/src/utils/cached_stream_controller.dart';
+import 'package:matrix/src/voip/utils/stream_helper.dart';
+
+/// Wrapped MediaStream, used to adapt Widget to display
+class WrappedMediaStream {
+  MediaStream? stream;
+  final Participant participant;
+  final Room room;
+
+  /// Current stream type, usermedia or screen-sharing
+  String purpose;
+  bool audioMuted;
+  bool videoMuted;
+  final Client client;
+  final bool isWeb;
```

Comment:
We need this, because we can't depend on flutter?

Reply:
Yes, that would be better, since it would also be a constant allowing for better code generation.

Reply:
done